src/pycell.rs

```diff
@@ -372,7 +372,7 @@ impl<T: PyClass> AsPyPointer for PyCell<T> {
     }
 }
 
-impl<T: PyClass> ToPyObject for &PyCell<T> {
+impl<T: PyClass + Send + Sync> ToPyObject for &PyCell<T> {
```

Comment:
As discussed in #948 I don't think we don't need these bounds. Let's keep it as it was?

```suggestion
impl<T: PyClass> ToPyObject for &PyCell<T> {
```